showpass/Showpass-E2E-tests
# Verify 'event-2' details as part of TA-39

**Description:**
When end users search for and open an event they are able to see the event page with all the details on it.
This spec will be verifying that an event can be found, selected and that the event details properly show up.
Steps:
- Navigate to [Create Amazing Experiences](https://beta.showpass.com/) 
- Enter an existing event name into the search input field
- Wait for the search result drop-down modal
- Select the event
- Make sure the event page properly shows up with all the event details

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-39

**How to test:**
Run the following spec and ensure the tests pass
cypress/e2e/events/verify-events.cy.js

**Test execution results:**

<img width="774" alt="TA-39-part-2-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/94be6014-190c-46fd-ba94-6aaae3f66afa">
